server.js

```diff
@@ -33,7 +33,7 @@ function handleRedirectFor(urls, url, res) {
 function handleNonWwwToWwwRedirect(req, res) {
   try {
     const host = req.header('host');
-    if (!host.match(/^www\..*/i)) {
+    if (!host.match(/^www\..*/i) && host.match(/^globalforestwatch\..*/i)) {
```

Comment:
Only change I'd potentially make would be to move this down to line 56 

```
if (process.env.NEXT_PUBLIC_FEATURE_ENV === 'production') {
  handleNonWwwToWwwRedirect(req, res);
}
```

where we check whether we should apply the redirect, and add the why in the comments.

Reply:
OK, it makes sense!